apps/condo/domains/billing/components/ServicesModal.tsx

```diff
@@ -99,17 +98,20 @@ export const ServicesModal: React.FC<IServicesModalProps> = ({
     const accountNumber = get(receipt, ['account', 'number'])
     const address = get(receipt, ['property', 'address'])
     const unitName = get(receipt, ['account', 'unitName'])
+    const unitType = get(receipt, ['account', 'unitType'])
+
+    const UnitTypePrefix = intl.formatMessage({ id: `field.UnitType.prefix.${unitType}` }).toLocaleLowerCase()
```

Comment:
Do you really need to use to lowe?

Reply:
We have the same way in address display from TicketChange.  So, this was done to keep consistency
```js
const addressChangePostfix += `, ${intl.formatMessage({ id: `field.UnitType.prefix.${unitType}` }).toLowerCase()}. ${unitName}`
```